.github/workflows/bootstrap.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Lint plugin_template
         working-directory: plugin_template
         run: |
-          pip3 install black flake8
+          pip3 install black flake8 "click<8.1"
```

Comment:
Wonderful. Can you put that link in the comment of this line and also the line below it. Something like `Unpin click when <link here> is resolved.